.github/workflows/stage-4-acceptance.yaml

```diff
@@ -172,6 +194,21 @@ jobs:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.AWS_ASSUME_ROLE_NAME }}
           role-session-name: deployInfra
           aws-region: eu-west-2
-      - name: "Destroy sandbox"
+  sandbox-tear-down:
+    name: "Sandbox tear down"
+    if: success() || failure()
+    runs-on: ubuntu-latest
+    needs: [test-accessibility, test-ui-component]
+    environment: dev
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v4
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.AWS_ASSUME_ROLE_NAME }}
+          role-session-name: deployInfra
+          aws-region: eu-west-2
+      - name: "Destroy Backend sandbox"
         run: |
-          npm run destroy-sandbox -- --identifier "wf-${GITHUB_RUN_ID}"
+          npm run destroy-backend-sandbox wf-${GITHUB_RUN_ID}
```

Comment:
you appear to have lost the teardown command for the Amplify sandbox

Reply:
good spot - fixed